tests/components/recorder/test_init.py

```diff
@@ -634,6 +638,7 @@ def test_recorder_setup_failure(hass):
 
 def test_recorder_setup_failure_without_event_listener(hass):
```

Comment:
It would be good to validate we still get the frontend when the recorder setup fails in this case